.gitignore

```diff
@@ -0,0 +1,3 @@
+target/
+.envrc
```

Comment:
i don't think .envrc should be ignored, we use it to set up direnv in the top level directory

Reply:
.envrc should not be .gitignored

Reply:
.envrc should NOT be .gitignored